data-prepper-plugins/lambda/src/main/java/org/opensearch/dataprepper/plugins/common/lambda/accumlator/Buffer.java

```diff
@@ -3,9 +3,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package org.opensearch.dataprepper.plugins.sink.lambda.accumlator;
+package org.opensearch.dataprepper.plugins.common.lambda.accumlator;
```

Comment:
Please use the package `org.openserach.dataprepper.plugins.lambda.` for all classes.

Thus, this would become:

```
org.opensearch.dataprepper.plugins.lambda.common.accumlator;
```

Reply:
I have followed the same pattern as opensearch plugin. i can make the suggested changes, but just want to make sure the pattern followed in the package is consistent.

Reply:
Yes, the pattern I described above should be the preferred one.

```
org.opensearch.dataprepper.plugins.<plugin>.<anything_for_that_plugin>
```